UnchainedLauncher.GUI/ViewModels/Registry/LocalModRegistryWindowVM.cs

```diff
@@ -0,0 +1,303 @@
+using CommunityToolkit.Mvvm.Input;
+using LanguageExt;
+using PropertyChanged;
+using Semver;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Windows;
+using UnchainedLauncher.Core.JsonModels.Metadata.V3;
+using UnchainedLauncher.Core.Services.Mods.Registry;
+using UnchainedLauncher.Core.Utilities;
+using UnchainedLauncher.GUI.Views.Registry;
+
+namespace UnchainedLauncher.GUI.ViewModels.Registry {
+    using static LanguageExt.Prelude;
+
+    public class ReleaseCreationHelper {
+        public LocalModRegistry Registry { get; set; }
+
+        public ReleaseCreationHelper(LocalModRegistry registry) {
+            Registry = registry;
+        }
+
+        public RegistryReleaseFormVM PromptAddRelease(Release newRelease, string? previousPakPath = null) {
+            var editableForm = new RegistryReleaseFormVM(newRelease, previousPakPath);
+            var editWindow = new RegistryReleaseFormWindow();
```

Comment:
ViewModels shouldn't explicitly spawn windows like this because it tightly couples the ViewModel to the window.  Windows can be tightly coupled to view models, but not vice versa